release-notes/v2.1.0.md

```diff
@@ -0,0 +1,36 @@
+# HPE CSI Driver for Kubernetes 2.1.0 Release Notes
+
+## HPE CSI Driver for Kubernetes 2.1.0
+
+| **Version:** |2.1.0|
+|--------------|-----|
+| **Revision:** | Tuesday January 10, 2022 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.1.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+
+The following table lists the resolved issues for HPE Alletra 9000 and Primera/3PAR CSP for HPE CSI Driver release 2.1.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v2.1.0. Please note that, `Known Issues` from previous releases are still applicable with suggested workarounds, if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+
+
+The following table lists the known issues for HPE Alletra 9000 and Primera/3PAR CSP for HPE CSI Driver release 2.1.0
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-2330|csi.k8s|Discrepancy in volume size between array and master. | When volume size is not provided in multiple of 256 then array round it off to nearby 256 multiple value but on the node level it's size get not reflected. |None|
+|CON-2536|csi.k8s| RCG Greater than 23 characters generating errors. | When user provides RCG name more than 23 characters, array does not create rcg as it is against it's core behaviour |RCG name should always be less than 23 characcters. |
```

Comment:
Description: Volume mutation fails if the RCG name specified is invalid or longer than 22 characters. There is no feedback given to the end user about the invalid input, except in logs.
Workaround: User can mutate the PVC again providing a valid RCG name (should be less than or equal to 22 characters and should not contain special characters except ..TBD: please specify what it should contain)

Reply:
modified the same content except the special characters which is to be decided